x-pack/dev-tools/jest/create_jest_config.js

```diff
@@ -59,6 +59,7 @@ export function createJestConfig({ kibanaDirectory, rootDir, xPackKibanaDirector
       `${kibanaDirectory}/src/dev/jest/setup/react_testing_library.js`,
     ],
     testMatch: ['**/*.test.{js,ts,tsx}'],
+    testPathIgnorePatterns: ['integration_tests/'],
```

Comment:
The normal jest config should not run the integration tests. This is consistent with the OSS jest config.